static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-ticker.js

```diff
@@ -1,19 +1,19 @@
 // @flow
 
 export const acquisitionsEpicTickerTemplate = `
-    <div class="epic-ticker">
-        <div class="epic-ticker__so-far">
-            <div class="epic-ticker__count">$0</div>
+    <div id="epic-ticker" class="js-epic-ticker epic-ticker">
+        <div class="js-ticker-so-far epic-ticker__so-far">
+            <div class="js-ticker-count epic-ticker__count">$0</div>
```

Comment:
these template files should probably have 'US' in the name since they're hardcoded to dollars

Reply:
great point!